lib/iris/cube.py

```diff
@@ -2682,6 +2696,7 @@ def _intersect(
         min_inclusive=True,
         max_inclusive=True,
         ignore_bounds=False,
+        threshold=0,
```

Comment:
I keep thinking about this and I'm not sure to be honest. Of course more comments generally are a good thing, but I'm not convinced that what's going on here is surprising enough to bother? At least to the kind of person likely to look through this code.

Though I may be setting the bar for that higher than normal because such comment would need to go in the argument list, which just feels awkward.

Reply:
I'm happy for you to make that call either way - I agree that people are likely to figure out what's going on.

I was a bit surprised as I worked out how the code works that it's normally used by expanding a tuple across the args and the first two kwargs (probably more so than threshold being repeated) but I don't think that's necessarily odd enough for a comment.